cmd/configure.go

```diff
@@ -93,7 +93,11 @@ func runConfigure(configuration config.Configuration, flags *pflag.FlagSet) erro
 
 		ok, err := client.IsPingable()
 		if !ok || err != nil {
```

Comment:
The IsPingable method seems to imply if there is an error than the client is not technically not reachable. If that is always going to be the case should we just check if `err != nil` and drop the `!ok`?

Reply:
Yeah, now that the IsPingable() returns an error, we can simplify.